README.md

```diff
@@ -1,2 +1,51 @@
-# funding-service-design-post-award-frontend
-funding-service-design-post-award-frontend
+
+## Prerequisites
+
+### Required
+
+- Python 3.8.x or higher
```

Comment:
sorry to chip in on the review late on but could this be up-versioned? I assume we're going to run on Python 3.11 like the backend

Reply:
this may generate new requirements files I suppose so may be worth running 
```
    pip-compile requirements.in

    pip-compile requirements-dev.in
```
afterwards if you were working on 3.8 before?